server/service/integration_core_test.go

```diff
@@ -12292,3 +12292,108 @@ func (s *integrationTestSuite) TestHostWithNoPoliciesClearsPolicyCounts() {
 	require.Len(t, listHostsResp.Hosts, 1)
 	require.Equal(t, uint64(0), listHostsResp.Hosts[0].FailingPoliciesCount)
 }
+
+func (s *integrationTestSuite) TestHostSoftwareWithTeamIdentifier() {
+	t := s.T()
+	ctx := context.Background()
+
+	host, err := s.ds.NewHost(ctx, &fleet.Host{
+		NodeKey:       ptr.String(t.Name()),
+		OsqueryHostID: ptr.String(t.Name()),
+		UUID:          t.Name(),
+		Hostname:      t.Name() + "foo.local",
+		Platform:      "darwin",
+	})
+	require.NoError(t, err)
+
+	safariApp := fleet.Software{
+		Name:             "Safari.app",
+		BundleIdentifier: "com.apple.safari",
+		Version:          "18.1",
+		Source:           "apps",
+	}
+	googleChromeApp := fleet.Software{
+		Name:             "Google Chrome.app",
+		BundleIdentifier: "com.google.Chrome",
+		Version:          "130.0.6723.117",
+		Source:           "apps",
+	}
+	ghCli := fleet.Software{
+		Name:   "gh",
+		Source: "homebrew_packages",
```

Comment:
Homebrew packages won't have a team identifier?

Reply:
Good question. I haven't seen signed bundles in homebrew packages:
```sh
find /usr/local/Cellar/ -name \*.app -exec codesign --display --verbose {} \;
/usr/local/Cellar//python@3.9/3.9.19_1/Python Launcher 3.app: code object is not signed at all
[...] // all output with code object is not signed at all
```

Reply:
Just ran on my system and that matches what I'm seeing. Fair enough. We'll need to note that we skip Homebrew somewhere.

Reply:
We are documenting that this information is only returned for macOS .app applications: https://github.com/fleetdm/fleet/pull/23743, let me know if we should clarify a little more.